src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -720,7 +721,7 @@ private void EnsureSessionHandleExists(WinHttpRequestState state)
                         if (sessionHandle.IsInvalid)
                         {
                             int lastError = Marshal.GetLastWin32Error();
-                            WinHttpTraceHelper.Trace("WinHttpHandler.EnsureSessionHandleExists: error={0}", lastError);
+                            if (NetEventSource.IsEnabled) NetEventSource.Info(this, $"error={lastError}");
```

Comment:
Why is this Info rather than Error?  I'm just curious how you're deciding when to use which.

Reply:
That is a mistake.  I'll change that to Error.